### PR TITLE
feat: update version of docsy to 03eede2 

### PR DESCRIPTION
**Pull request by h1z3y3**
I'm getting into the Chinese translation, but found some bugs of docsy, so I update it to latest version.

There were some updates of docsy, like this added some i18n message for zh.toml, it's necessary for Chinese:
https://github.com/google/docsy/pull/826/files

## What updated:

1. update the docsy to the latest version: [19e0072b](https://github.com/google/docsy/commit/19e0072b)
2. import docsy as hugo module, for more information: https://github.com/google/docsy/discussions/535, and this release of docsy is the last release with submodule-only support: https://github.com/google/docsy/releases
3. fix duplicate `zh*.toml` i18n file, this is my PR in docsy: https://github.com/google/docsy/pull/832

### CODE REVIEW

**Comment by h1z3y3** [comment]
@myitcv  
After a few days this PR was still open, 
I've updated theme to the  `theme/docsy` latest version.
Bring some good features and fix some i18n bugs, please help me to review 😊

File: config/_default/config.toml [hunk 1]
```diff
@@ -181,3 +181,13 @@ navbar_logo = false
 	url = "https://review.gerrithub.io/plugins/gitiles/cue-lang/cue"
 	icon = "fab fa-git"
         desc = "The source-of-truth repo."
+[module]
```
Comment (myitcv) [inline config/_default/config.toml:184]: Why is this change needed? 

Is this part of a switch to docsy using Go modules as opposed to git sub modules?

If so, does the submodule itself need to be removed now?

**Review by myitcv — COMMENTED** [review]
> I'm getting into the Chinese translation, but found some bugs of docsy, so I update it to latest version.

Wonderful, thank you for taking the initiative here! 

Got some questions about the change. 

Following this PR I also can't develop things locally using the version configured in:

https://github.com/cue-lang/cuelang.org/blob/2666f043f626aa97c80551c1175f80fe2c5a525e/netlify.toml#L10

It hangs on:

```
$ huge serve -D
hugo: downloading modules …
```

Does that now need to change?

File: config/_default/config.toml (hunk 1 above)
Comment (myitcv) [reply config/_default/config.toml]: If moving to v0.1.0 works for you, that seems like a better change to make now. 

We can of course revisit upgrading further in a later PR.

**Comment by myitcv** [comment]
@h1z3y3 - thanks for your patience on this PR. Now merged.

**Comment by h1z3y3** [comment]
I'm the one to say thank you. 
Thank you for taking CUE to the world. 

I will do further work when docsy go module version is stable.

**Comment by myitcv** [comment]
Thanks very much! Wonderful that you are taking the initiative here.

If you are starting work on translations, perhaps you could comment on via https://github.com/cue-lang/cue/issues/970?

That way others can help review, contribute etc?

**Comment by h1z3y3** [comment]
Okay. I'm working on translation of https://curatorials.com and I have finished some pages. It's also an awesome website introducing CUE. 

I'll start on cuelang.org later. 

But what obstructs me is that docsy i18n still has some bugs on Chinese version. 

It will be soon, don't worry. Thanks
